lib/searchkick/query.rb

```diff
@@ -191,7 +191,7 @@ def handle_response(response)
     end
 
     def retry_misspellings?(response)
-      @misspellings_below && Results.new(searchkick_klass, response).total_count < @misspellings_below
+      @misspellings_below && response["error"].blank? && Results.new(searchkick_klass, response).total_count < @misspellings_below
```

Comment:
It might make more sense to check `response["hits"].present?`, rather than `response["error"].blank?`. I think that which to choose is mostly a matter of taste/philosophy, and I defer to yours, of course, @ankane (or maybe there's some other approach altogether that you think would be better to avoid the exception that we were able to surface).

Reply:
On this topic, should we technically be calling either of the `activesupport` methods `present?` or `blank?`? I noticed that there are already places in the code base where we are calling both of these methods, and that works fine, for now, I think, because I think that `searchkick` must be used with either `activerecord` or `mongoid`, which I think in turn ultimately require `active_support/core_ext/object/blank`. However, I think it's _theoretically_ possible that either of those dependencies might stop requiring `active_support/core_ext/object/blank`, in which case we might start getting exceptions in `searchkick` where we call `present?` or `blank?`.

I think that this theoretical issue could be resolved by adding `activesupport` as an explicit dependency in `searchkick.gemspec`, and then explicitly requiring `active_support/core_ext/object/blank` in `lib/searchkick.rb`.

Reply:
A simpler alternative (versus adding `activesupport` as a dependency) would be to change this to `response["error"].nil?`. I will make that change now.